util/EFSOI_Utilities/src/gridio_efsoi.f90

```diff
@@ -0,0 +1,898 @@
+         ! 
+         call copytogrdin(ug,tv(:,k))
+		 
+         ! Transformation to EFSOI relevant quantities
+         ! Mass component quantities
+         tv(:,k) = cptr * weight(:,k) * tv(:,k)
+         ! Moisture component transormation to EFSOI
+         ! relevant quantities
+         q(:,k) = qweight * weight(:,k) * q(:,k)
+         tv_to_t(:,k) = ( one / (one + q(:,k)*0.61_r_kind) )
+    
+         ! Approximate the necessary transformation
+         ! of virtual temperature to temperature
+         tv(:,k) = tv(:,k) * tv_to_t(:,k)
+		 		 
+         if (tv_ind > 0)   grdin(:,levels(tv_ind-1)+k) = tv(:,k)
+         if (q_ind > 0)    grdin(:,levels( q_ind-1)+k) =  q(:,k)
+      enddo
+	  
+      deallocate(ug3d,vg3d)
+	  
+  else
+	  
+	 ! $omp parallel do private(k,ug,vg,divspec,vrtspec) 
+	 ! shared(sigdata,pressi,vmassdiv,grdin,tv,q,cw,u_ind,v_ind,pst_ind,q_ind,tsen_ind,cw_ind,qi_ind,ql_ind)
+	 do k=1,nlevs
+
+	    vrtspec = sigdata%z(:,k); divspec = sigdata%d(:,k)
+	    call sptezv_s(divspec,vrtspec,ug,vg,1)
+	    if (u_ind > 0) then
+	       call copytogrdin(ug,grdin(:,levels(u_ind-1)+k))
+	    endif
+	    if (v_ind > 0) then
+	       call copytogrdin(vg,grdin(:,levels(v_ind-1)+k))
+	    endif
+
+	    ! calculate vertical integral of mass flux div (ps tendency)
+	    ! this variable is analyzed in order to enforce mass balance in the analysis
+	    !if (pst_ind > 0) then
+	    !   ug = ug*(pressi(:,k)-pressi(:,k+1))
+	    !   vg = vg*(pressi(:,k)-pressi(:,k+1))
+	    !   call sptezv_s(divspec,vrtspec,ug,vg,-1) ! u,v to div,vrt
+	    !   call sptez_s(divspec,vmassdiv(:,k),1) ! divspec to divgrd
+	    !endif
+
+	    divspec = sigdata%t(:,k)
+	    call sptez_s(divspec,ug,1)
+	    call copytogrdin(ug,tv(:,k))
+	    
+	    divspec = sigdata%q(:,k,1)
+	    call sptez_s(divspec,vg,1)
+	    call copytogrdin(vg,q(:,k))
+		
+        ! Transformation to EFSOI relevant quantities
+        ! Mass component quantities
+        tv(:,k) = cptr * weight(:,k) * tv(:,k)
+        ! Moisture component transormation to EFSOI
+        ! relevant quantities
+        q(:,k) = qweight * weight(:,k) * q(:,k)
+        tv_to_t(:,k) = ( one / (one + q(:,k)*0.61_r_kind) )
+   
+        ! Approximate the necessary transformation
+        ! of virtual temperature to temperature
+        tv(:,k) = tv(:,k) * tv_to_t(:,k)
+				
+		if (tv_ind > 0)          grdin(:,levels(tv_ind-1)+k) = tv(:,k)
+	    if (q_ind > 0)           grdin(:,levels( q_ind-1)+k) =  q(:,k)	  	  
+	  
+     enddo
+  endif
+  	  	  
+
+  ! =============================================================
+  ! surface pressure 
+  ! ================
+  if (ps_ind > 0) then
+    call copytogrdin(psg,grdin(:,levels(n3d) + ps_ind))
+  endif
+
+  ! Transformation to EFSOI relevant quantities
+  ! Surface pressure contribution
+  grdin(:,levels(n3d) + ps_ind) = rdtrpr * grweight(:) * 100._r_kind * grdin(:,levels(n3d) + ps_ind)
+
+
+
+  
+  deallocate(psg)
+ 
+  ! updated by LL
```

Comment:
@AndrewEichmann-NOAA Comments with author initials should not be included in the code itself (there are a few instances in this code).  Developer acknowledgement should only be in the top documentation block.

Reply:
done, removed all in file, found no more instances of LL in this PR